lib/screens/siteConfig/CertificateScreen.dart

```diff
@@ -219,34 +219,22 @@ class _CertificateScreenState extends State<CertificateScreen> {
           ConfigButtonItem(
               content: Center(child: Text('Choose a file')),
               onPressed: () async {
-                var file;
                 try {
-                  await FilePicker.clearTemporaryFiles();
-                  file = await FilePicker.getFile();
-
-                  if (file == null) {
-                    print('GOT A NULL');
+                  final content = await Utils.pickFile(context);
+                  if (content == null) {
                     return;
                   }
-                } catch (err) {
-                  print('HEY $err');
-                }
 
-                var content = "";
-                try {
-                  content = file.readAsStringSync();
+                  _addCertEntry(content, (err) {
+                    if (err != null) {
+                      Utils.popError(context, 'Error loading certificate file', err);
```

Comment:
Should this return? Previously this was the last line of code in this function - now the catch comes after it. I could see someone adding code following the catch, and not realizing that this doesn't return in the case of an error.

Reply:
Ignore me - this is occurring as part of a callback. The order confused me a bit.